pkg/topologies/smokev1/smokev1.go

```diff
@@ -184,6 +185,9 @@ func (s *SmokeRouterOnlyTopology) initializeInteriors() {
 
 	// Initializing the interior routers
 	s.IcInteriorEast = s.defaultInteriorSpec()
+    if os.Getenv("IMAGE_QDROUTERD_INTEROP") != "" {
+        s.IcInteriorEast.DeploymentPlan.Image = os.Getenv("IMAGE_QDROUTERD_INTEROP")
+    }
```

Comment:
Same here... just running a `go fmt`, should help, I believe.

Reply:
fixed